packages/lib-classifier/src/plugins/tasks/survey/components/components/Chooser/components/Choices/components/ChoiceButton.js

```diff
@@ -66,52 +56,86 @@ function ChoiceButton({
     }
   })
 
-  let thumbnailHeight = 0
-  if (thumbnailSize === 'small') {
-    thumbnailHeight = 21
-  }
-  if (thumbnailSize === 'medium') {
-    thumbnailHeight = 42
-  }
-  if (thumbnailSize === 'large') {
-    thumbnailHeight = 84
-  }
-
-  const thumbnailWidth = Math.round(thumbnailHeight * THUMBNAIL_ASPECT_RATIO)
-  const thumbnailSrc = `https://thumbnails.zooniverse.org/${thumbnailWidth}x${thumbnailHeight}/${src.slice(8)}`
+  const background = selected
+    ? 'neutral-1'
+    : shadedBackground
+      ? {
+          dark: 'dark-4',
+          light: 'light-1'
+        }
+      : {
+          dark: 'dark-5',
+          light: 'neutral-6'
+        }
+  const border = selected
+    ? { color: 'neutral-6', size: '1px' }
+    : undefined
+  const textColor = selected 
+    ? 'neutral-6' 
+    : {
+        dark: 'neutral-6',
+        light: 'neutral-7'
+      }
+  
+  const thumbnailWidth = Math.round(THUMBNAIL_ASPECT_RATIO * 50)
+  const thumbnailSrc = `https://thumbnails.zooniverse.org/${thumbnailWidth}x50/${src.slice(8)}`
```

Comment:
Increase width by 2 or 3 times (100px) for better quality, can likely drop height.